app/views/versions/index.html.erb

```diff
@@ -38,17 +38,22 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= render partial: 'versions/overview', locals: {version: version} %>
       <%= render(partial: "wiki/content", locals: {content: version.wiki_page.content}) if version.wiki_page %>
       <% if (issues = @issues_by_version[version]) && issues.size > 0 %>
-        <strong class="related-issues-heading"><%= l(:label_related_work_packages) %></strong>
-        <%= form_tag({}) do -%>
-          <table class="list related-issues">
-            <%- issues.each do |issue| -%>
-              <tr class="hascontextmenu">
-                <td class="checkbox"><%= check_box_tag 'ids[]', issue.id %></td>
-                <td><%= link_to_work_package(issue, project: (@project != issue.project)) %></td>
-              </tr>
-            <%- end -%>
-          </table>
-        <% end %>
+      <%= form_tag({}) do -%>
```

Comment:
If you don't need any magic in the form tag such as methods, request paths etc., you can just use a `<form>` tag and save the extra method call.